src/components/toast/toast.js

```diff
@@ -213,7 +213,9 @@ function MdToastProvider($$interimElementProvider) {
         var opts = {
           template: [
             '<md-toast md-theme="{{ toast.theme }}" ng-class="{\'md-capsule\': toast.capsule}">',
-              '<span flex>{{ toast.content }}</span>',
+              '<span flex role="alert" aria-relevant="all" aria-atomic="true">' +
```

Comment:
This won't actually work, because the Toast is inserted and then removed–ARIA Live Regions have to be present at load time and remain in the page to be picked up by screen readers. We talked a while back about creating a global messaging directive for this purpose–could that be incorporated here? https://github.com/angular/material/issues/3174